clients/lab-sensible-terminal

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
```

Comment:
This might need a License to make distros happy. Suggest something very broad like the Unlicence or MIT / BSD.

Reply:
I think I prefer sticking with GPL2 to avoid having multiple licenses. What would be the arguments for MIT/BSD?

Reply:
Might make re-using / forking the script easier. Although now thinking of it, this is not a library or anything like that so using something broad vs GPL2 shouldn't really make a difference in this case.

Reply:
Have added SPDX GPL-2.0-only identifier in file.